application/actions/user_test.go

```diff
@@ -225,19 +225,39 @@ func (as *ActionSuite) TestUpdateUser() {
 		},
 		{
 			Name: "not allowed",
-			Payload: fmt.Sprintf(`mutation {updateUser(input:{id: \"%v\", location: \"%v\"}) {nickname}}`,
+			Payload: fmt.Sprintf(`mutation {user: updateUser(input:{id: "%v", location: %v}) {nickname}}`,
 				f.Users[0].UUID, location),
 			TestUser:    f.Users[1],
 			Test:        func(t *testing.T) {},
-			ExpectError: true,
+			ExpectError: "not allowed",
+		},
+		{
+			Name: "remove photo",
+			Payload: fmt.Sprintf(`mutation {user: updateUser(input:{id: "%v", location: %v}) {avatarURL}}`,
+				f.Users[1].UUID, location),
+			TestUser: f.Users[0],
+			Test: func(t *testing.T) {
+				as.Equal(f.Users[1].AuthPhotoURL.String, resp.User.AvatarURL, "expected photo to be deleted")
```

Comment:
It's a bit misleading because it's a tiered response. If no attached photo, then it uses the auth photo, if no auth photo, then gravatar. Should never be "".